help/add-a-custom-linkifier.md

```diff
@@ -31,28 +31,32 @@ This is best explained by example.
 Hash followed by a number of any length.
 
 * Pattern: `#(?P<id>[0-9]+)`
-* URL format string: `https://github.com/zulip/zulip/issues/%(id)s`
+* URL Template: `https://github.com/zulip/zulip/issues/{id}`
 * Original text: `#2468`
 * Automatically links to: `https://github.com/zulip/zulip/issues/2468`
 
 String of hexadecimal digits between 7 and 40 characters long.
 
 * Pattern: `(?P<id>[0-9a-f]{7,40})`
-* URL format string: `https://github.com/zulip/zulip/commit/%(id)s`
+* URL Template: `https://github.com/zulip/zulip/commit/{id}`
 * Original text: `abdc123`
 * Automatically links to: `https://github.com/zulip/zulip/commit/abcd123`
 
 Generic GitHub `org/repo#ID` format:
 
 * Pattern: `(?P<org>[a-zA-Z0-9_-]+)/(?P<repo>[a-zA-Z0-9_-]+)#(?P<id>[0-9]+)`
-* URL format string: `https://github.com/%(org)s/%(repo)s/issues/%(id)s`
+* URL Template: `https://github.com/{org}/{repo}/issues/{id}`
 * Original text: `zulip/zulip#2468`
 * Automatically links to: `https://github.com/zulip/zulip/issues/2468`
 
 Linkifier patterns are regular expressions, using the
 [re2](https://github.com/google/re2/wiki/Syntax) regular expression
 engine.
 
+Linkifiers use [RFC 6570](https://www.rfc-editor.org/rfc/rfc6570.html)
+compliant URL templates to represent the link to be generated from the
+pattern.
```

Comment:
This probably should appear a bit earlier in the page.

Reply:
Moved it to the intro part.